tools/ci-build/README.md

```diff
@@ -0,0 +1,39 @@
+ci-build
+========
+
+This directory includes the build Docker image and scripts to use it in CI.
+- `acquire-build-image`: Script that retrieves the build image from public ECR or builds one locally
+  depending on the state of the tools directory. If the tools have changed (in git history), then it opts
+  to build a new image rather than reuse an existing one.
+- `add-local-user.dockerfile`: Creates a user in the build image with the host's user ID
+- `build-image.dockerfile`: Dockerfile used to create the build image
```

Comment:
not required, but a more typical design would probably be a directory with a dockerfile in it instead of named dockerfiles

```
base-image/Dockerfile
add-local-user/Dockerfile
...
```
this is because there is tooling around a dockerfile being named `Dockerfile`